testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -57,99 +58,85 @@ def add_item(
             item["priority"] = priority
         self.section.append(item)
 
+    @modify
+    def clear_all(self):
+        """Removes all identities from AuthConfig"""
+        self.section.clear()
 
-class Identities(Section):
+
+class IdentitySection(Section):
     """Section which contains identity configuration"""
 
     @modify
-    def mtls(self, name: str, selector_key: str, selector_value: str, **common_features):
+    def add_mtls(self, name: str, selector: Selector, **common_features):
```

Comment:
changing number of params here is a breaking change

Reply:
everything in this PR is a breaking change :)